google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -227,7 +234,12 @@ def bq_to_arrow_field(bq_field, array_type=None):
         if array_type is not None:
             arrow_type = array_type  # For GEOGRAPHY, at least initially
         is_nullable = bq_field.mode.upper() == "NULLABLE"
-        return pyarrow.field(bq_field.name, arrow_type, nullable=is_nullable)
+        metadata = BQ_FIELD_TYPE_TO_ARROW_FIELD_METADATA.get(
+            bq_field.field_type.upper() if bq_field.field_type else ""
```

Comment:
Looking at [pyarrow.field](https://arrow.apache.org/docs/python/generated/pyarrow.field.html#pyarrow.field) docs, `metadata` should be `dict` or `None`, but we sometimes use an empty string. Is there a reason for that?

I suppose it still works, but if nothing else a type checker might start complaining at some point when it realizes that.

Reply:
The empty string is just the argument to `get`. It's not metadata.  We only pass a `dict` or `None`.

Reply:
Meh, of course, and an empty string won't mach any keys, resulting in `None`. And I though I saw a comma somewhere making `""` a second argument. Nevermind. :)